tests/db_functions/test_abs.py

```diff
@@ -0,0 +1,42 @@
+from decimal import Decimal
+
+from django.db.models import DecimalField
+from django.db.models.functions import Abs
+from django.test import TestCase
+
+from .models import DecimalModel, FloatModel, IntegerModel
+
+
+class AbsTests(TestCase):
+
+    def test_abs_decimal(self):
```

Comment:
Please omit "_abs" and similar in all test names.

Reply:
Thanks. I have put all tests into a new folder tests/db_functions/math.